### ISSUE TITLE
Evaluate work needed for React a11y tool bump violations

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#12296]
The [patternfly-a11y version bump PR](https://github.com/patternfly/patternfly-react/pull/12202) flagged a11y violations for various components. The work needed to complete this issue involves:

1. Identifying what is causing the violations
2. Opening up followup issues for each component outlining what work needs to be done for that component, and adding it to the overall [Resolve violations from a11y tool bump Jira ticket](https://redhat.atlassian.net/browse/PF-3831)

---

**Jira Issue:** [PF-3837](https://redhat.atlassian.net/browse/PF-3837)

**Comment by kmcfaul** [comment]
Closing for follow up #12558
